assets/js/stats/utils.js

```diff
@@ -10,12 +10,20 @@ export function debounce( func, delay ) {
 	};
 }
 
-export function filterZeroes( list ) {
-	return list.filter( function ( i ) {
-		return i > 0;
-	} );
+export function getPostId( node ) {
+	return +node.className.match( /\bpost-(\d+)\b/ )?.[ 1 ] || 0;
```

Comment:
```suggestion
	return node.className.match( /\bpost-(\d+)\b/ )?.[ 1 ] || 0;
```

Reply:
The `+` is a cheeky little type cast to number :)